rasterio/plot.py

```diff
@@ -33,38 +33,69 @@
 logger = logging.getLogger(__name__)
 
 
-def show(source, cmap='gray', with_bounds=True):
+def show(source, cmap='gray', with_bounds=True, ax=None, title=None, **kwargs):
```

Comment:
not to start a round of bike-shedding, but I wonder if we shouldn't switch to the matplotlib default - `viridis` - if I'm not mistaken.  I'm biased by the PyCon lightning talk about replacing the previous default with this one, for reasons of perceptual uniformity.  See [example](http://matplotlib.org/examples/color/colormaps_reference.html).

Specific to rasterio, I was thinking that something that is NOT black to white would make it easier to see NODATA separately from the data range.